buildSrc/src/main/kotlin/org/cqfn/save/buildutils/DockerStackConfiguration.kt

```diff
@@ -17,7 +18,11 @@ fun Project.createStackDeployTask() {
         description = "Set project version in docker-compose file"
         doFirst {
             val newText = file("$rootDir/docker-compose.yaml.template").readLines()
-                .joinToString(System.lineSeparator()) { it.replace("{{project.version}}", versionForDockerImages()) }
+                .joinToString(System.lineSeparator()) {
+                    var tmp = it
+                    tmp = tmp.replace("{{project.version}}", versionForDockerImages())
+                    tmp.replace("{{profile}}", profile)
```

Comment:
```suggestion
                    it.replace("{{project.version}}", versionForDockerImages())
                        .replace("{{profile}}", profile)
```
I think temporary variable is not needed

Reply:
Fixed